src/main/java/net/snowflake/ingest/streaming/example/SnowflakeStreamingIngestExample.java

```diff
@@ -47,16 +51,51 @@ public static void main(String[] args) throws Exception {
       // Open a streaming ingest channel from the given client
       SnowflakeStreamingIngestChannel channel1 = client.openChannel(request1);
 
+      final int totalRowsInTable = 1000;
+
       // Insert a few rows into the channel,
-      for (int val = 0; val < 1000; val++) {
+      for (int val = 0; val < totalRowsInTable; val++) {
         Map<String, Object> row = new HashMap<>();
+
+        // c1 corresponds to the column name in table
         row.put("c1", val);
-        InsertValidationResponse response = channel1.insertRow(row, null);
+
+        // Along with row, we are also passing in offset Token which corresponds to the row number
```

Comment:
Rather than annotating every few lines, maybe refactor the example such that it has different phases of loading and polling and annotate the top of each section with the explanation as to why the code does what it does?

Reply:
Will do!